test/metabase/search/postgres/index_test.clj

```diff
@@ -0,0 +1,120 @@
+(ns metabase.search.postgres.index-test
+  (:require
+   [clojure.test :refer [deftest is testing]]
+   [metabase.search.config :as search.config]
+   [metabase.search.impl :as search.impl]
+   [metabase.search.postgres.index :as search.index]
+   [metabase.search.postgres.ingestion :as search.ingestion]
+   [metabase.test :as mt]
+   [toucan2.core :as t2]))
+
+(set! *warn-on-reflection* true)
+
+(defn is-postgres?
```

Comment:
I expected to find something for matching on this already, but didn't spot it.

When we wire this up to the API this can move out of test.

Reply:
not a shorthand but maybe it's enough? `(= :postgres (metabase.db/db-type))`

Reply:
Perfect 👍